components/dashboard/src/repositories/detail/general/ConfigurationName.tsx

```diff
@@ -24,8 +23,9 @@ type Props = {
 export const ConfigurationNameForm: FC<Props> = ({ configuration }) => {
     const { toast } = useToast();
     const updateProject = useUpdateProject();
-    const [projectName, setProjectName, isNameDirty] = useDirtyState(configuration.name);
+    const [projectName, setProjectName] = useState(configuration.name);
 
+    const nameChanged = useMemo(() => projectName !== configuration.name, [projectName, configuration.name]);
```

Comment:
Not a blocker by any means, and more of a nitpick, but for simple derived values like this that result in a primitive there's usually not a benefit to memoize it compared to an inline derived value like `nameChanged = projectName !== configuration.name`. I usually use the following litmus test to decide if useMemo is helpful:

* Is the calculation expensive? If so, `useMemo()`
* If we're composing an object to pass as a prop, `useMemo()` to avoid uneccessary renders since a reference to an object changes each render if you define it inline.

I'm sure there's other cases it's helpful, but figured I'd share the situations I've found it useful vs. not useful.